creator/Documents/ScoreboardIntroduction.md

```diff
@@ -82,62 +82,62 @@ This command returns a list of all of the objectives in the world.
 To remove an objective from the game use:
 
 ```
-/scoreboard objective remove [objective name]
+/scoreboard objective remove <objective: string>
 ```
 
-### setdisplay list
+### setdisplay 
 
-Use the **list** option of **setdisplay** to see scoreboard information listed on the pause screen. You can specify whether the players are ranked in ascending or descending order, according to score.
+You can modify how scoreboard objectives are displayed in the game. There are three slots that an objective can be displayed in: **belowName**, **list**, and **sidebar**. Specifying a slot without an objective name clears that display slot.
```

Comment:
I tweaked how the **setdisplay** section is laid out. The same general information applies to each of the three arguments, so I figured talking about that first made sense.